packages/nc-gui/lang/en.json

```diff
@@ -721,7 +721,7 @@
     "groupBy": "Group By",
     "addSubGroup": "Add subgroup",
     "shareBase": {
-      "label": "Share Base",
+      "label": "Share base",
```

Comment:
Overriding previous version. @dstala not sync to crowdin last time?

Reply:
i had force updated last time. not sure why its coming back